app/build.gradle

```diff
@@ -67,11 +67,11 @@ dependencies {
 
     //debug implementation
     //debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.1.60"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.6.10"
 
     //retrofit and live data
     implementation 'com.squareup.retrofit2:retrofit2:2.7.1'
-    implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+    implementation 'com.squareup.retrofit2:converter-gson:2.8.2'
```

Comment:
These versions don't seem to match the project, so I just used Ctrl + enter to get modification suggestions from Android studio